src/output.cpp

```diff
@@ -2101,7 +2101,7 @@ static void output_comment_multi_simple(chunk_t *pc)
       cmt_idx++;
 
       // 1: step through leading tabs and spaces to find the start column
-      if (line.size() == 0)
+      if (line.size() == 0 && (line_column < cmt.base_col || options::cmt_convert_tab_to_spaces()))
```

Comment:
Please break the `if `after the zero to get two lines

Reply:
Done, replicated formatting of other such conditions in the file.